src/nodes/math/MathNode.js

```diff
@@ -386,7 +386,7 @@ export const all = /*@__PURE__*/ nodeProxy( MathNode, MathNode.ALL );
  * @param {Node | Number} x - The parameter.
  * @returns {Node<bool>}
  */
-export const any = /*@__PURE__*/ nodeProxy( MathNode, MathNode.ANY );
+export const anyof = /*@__PURE__*/ nodeProxy( MathNode, MathNode.ANY );
```

Comment:
Clearly, this PR hasn't been tested once 😞 .

If you rename `any` to `anyof`, you also have to update the method chaining:
```js
addMethodChaining( 'any', anyof );
```